util/check_translation.py

```diff
@@ -256,6 +293,58 @@ def filter_print_timeline(text, missing_filter):
         print(text)
 
 
+def find_duplicate_translations(trans, locale):
+    if trans.get(locale, None):
+        for category in ["replaceSync", "replaceText", "~effectNames"]:
+            already_found = []
+            remove = []
+            for old in trans[locale].get(category, []):
+                if not old.lower() in already_found:
+                    already_found.append(old.lower())
+                else:
+                    remove.append(old)
+                    print(f'Found duplicate entry for "{old}" in "{category}"!')
+            for entry in remove:
+                del trans[locale][category][entry]
+        print()
+    return trans
+
+
+def print_not_used_translations(not_used_trans):
```

Comment:
I think it would be better to do this in a test rather than in check_translation.py, where you are already in javascript, and we could (potentially?) throw an error if a translation isn't used.

I feel like a little bit that the tests and find_missing_translations script have superseded check_translation.py.  I have been wondering a little if I should remove it.

What do you think about all of this?

Reply:
I like the output of check_translations a little bit more.
i usually used the check_translation part to do all my translation checks and only controlled them in the end by the find_missing_timeline_translation script.
I think mostly because i am not that familiar with adding my crude hacks as real tests.